### PR TITLE
Refactored LocationFinder and added translations

### PR DESCRIPTION
**Pull request by robinpiets**
Refactored LocationFinder and added translations

### CODE REVIEW

File: packages/client/src/components/Location/LocationFinder.tsx [hunk 1]
```diff
@@ -65,109 +65,110 @@ const LocationFinder: FunctionComponent<LocationFinderProps> = ({
...
-     Workaround is not giving any variables if the query should be skipped. */
-  const skip = !!(
-    houseNumberError ||
-    !houseNumberFull ||
-    !postalCode ||
-    !isValidPostalcode(postalCode) ||
-    postalCodeError
-  );
   const { loading, error: graphqlError, data } = useQuery(findAddress, {
-    variables: skip ? undefined : variables,
```
Comment (afjlambert) [inline packages/client/src/components/Location/LocationFinder.tsx:131]: Are you sure? Now we can have skip = true but still pass the variables. Is het bug actually fixed?

File: packages/client/src/components/Location/LocationFinder.tsx [hunk 2]
```diff
@@ -240,25 +238,28 @@ const LocationFinder: FunctionComponent<LocationFinderProps> = ({
...
           defaultValue={postalCode}
-          error={
-            !!postalCodeError ||
-            (showLocationNotFound && isValidPostalcode(postalCode))
-          }
+          error={!!(postalCode && postalCodeError) || showLocationNotFound}
           label={t("common.postalcode label")}
           name="postalCode"
-          onBlur={handleBlur}
+          onBlur={(e) => {
```
Comment (afjlambert) [inline packages/client/src/components/Location/LocationFinder.tsx]: unused arg?

**Review by afjlambert — APPROVED** [review]
Approved but please see minor remarks/questions.

File: packages/client/src/components/Location/LocationFinder.tsx (hunk 1 above)
Comment (robinpiets) [reply packages/client/src/components/Location/LocationFinder.tsx:131]: The bug looks fixed when upgrading `@apollo/client` to `3.3.12`.

File: packages/client/src/components/Location/LocationFinder.tsx (hunk 2 above)
Comment (robinpiets) [reply packages/client/src/components/Location/LocationFinder.tsx]: Nice catch!

**Review by svenjens — APPROVED** [review]

**Pull request merged**